mrp_multi_level/wizards/mrp_multi_level.py

```diff
@@ -758,8 +759,11 @@ def _mrp_final_process(self, mrp_areas):
 
         for product_mrp_area in product_mrp_area_ids:
             # Build the time-phased inventory
-            if self._exclude_from_mrp(
-                product_mrp_area.product_id, product_mrp_area.mrp_area_id
+            if (
+                self._exclude_from_mrp(
+                    product_mrp_area.product_id, product_mrp_area.mrp_area_id
+                )
+                or product_mrp_area.supply_method == "phantom"
```

Comment:
This condition should be included as the default condition in the `_exclude_from_mrp`method.